components/gantry/gantry.go

```diff
@@ -48,16 +48,40 @@ type Gantry interface {
 	referenceframe.InputEnabled
 
 	// Position returns the position in meters
+	//
+	//    myGantry, err := gantry.FromRobot(robot, "my_gantry")
+	//
+	//    // Get the current positions of the axes of the gantry in millimeters.
+	//    position, err := myGantry.Position(context.Background(), nil)
 	Position(ctx context.Context, extra map[string]interface{}) ([]float64, error)
 
 	// MoveToPosition is in meters
 	// This will block until done or a new operation cancels this one
+	//
+	//    myGantry, err := gantry.FromRobot(robot, "my_gantry")
+	//
+	//    // Create a list of positions for the axes of the gantry to move to. Assume in this example that the gantry is multi-axis, with 3 axes.
```

Comment:
[Test](https://github.com/viamrobotics/rdk/actions/runs/8743490939/job/23994141269?pr=3824) complains that this line is too long. 
```suggestion
	//    // Create a list of positions for the axes of the gantry to move to.
	//    // Assume in this example that the gantry is multi-axis, with 3 axes.
```